src/Application.vala

```diff
@@ -17,13 +17,13 @@
 public class AppCenter.App : Granite.Application {
     public const OptionEntry[] APPCENTER_OPTIONS =  {
         { "show-updates", 'u', 0, OptionArg.NONE, out show_updates,
-        "Display the Installed Panel", null},
+        N_("Display the Installed Panel"), null},
```

Comment:
"Display the installed applications" better?  I am not sure title case is required in this situation (check with Dan)

Reply:
Sentence case is probably correct here since it's more like a tooltip than a button label. We usually don't use articles in verbal phrases like this though and since this goes to the "Updates" view we probably should refer to it that way, so maybe "Display updates view" ?